tasks/eva_2156/deprecate_variant_from_contig.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+from copy import deepcopy
+from datetime import datetime
+
+import pymongo
+
+from ebi_eva_common_pyutils.config_utils import get_mongo_uri_for_eva_profile
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+logger = log_cfg.get_logger(__name__)
+
+
+def inactive_object(variant):
+    inactive_variant = deepcopy(variant)
+    inactive_variant['hashedMessage'] = inactive_variant.pop('_id')
+    return {
+        "eventType": "DEPRECATED",
+        "accession": variant['accession'],
+        "reason": "Removed: The contig on which the variant was mapped is not accessioned by INSDC.",
+        "inactiveObjects": [inactive_variant],
+        "createdDate": datetime.now()  # "2018-07-30T13:21:59.333Z"
```

Comment:
Just for the convenience of tracking, shall we have the same timestamp for the entire batch of contigs that are being deprecated?

Reply:
Changed it.